tools/glib.supp

```diff
@@ -1,3 +1,12 @@
+# This GLib suppressions file is known to be used at least by
```

Comment:
Sorry, I more meant things the other way around, the cockpit one should be canonical.  I was just noting a possible downstream consumer.